src/plugins/data_source/config.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema, TypeOf } from '@osd/config-schema';
+
+export const configSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: false }),
+  wrappingKeyName: schema.string({
+    minLength: 1,
+    maxLength: 100,
+    defaultValue: 'wrappingKeyName',
+  }),
+  wrappingKeyNamespace: schema.string({
+    minLength: 1,
+    maxLength: 100,
+    defaultValue: 'wrappingKeyNamespace',
+  }),
+  wrappingKey: schema.arrayOf(schema.number(), {
+    minSize: 32,
+    maxSize: 32,
+  }),
```

Comment:
why size is 32?

Reply:
could you resue `WRAPPING_KEY_SIZE: number = 32;`

Reply:
> why size is 32?

it's required for the encryption algo underneath